src/prpy/serialization.py

```diff
@@ -195,6 +216,17 @@ def serialize_transform(t):
     }
 
 # Deserialization.
+
+class UnitaryMemoizer:
```

Comment:
Missing no longer. (Is `UnitaryMemoizer` not sufficiently self-describing? `d-:`)